src/frontend/components/UI/AppVersion/index.tsx

```diff
@@ -1,16 +1,21 @@
 import React, { useEffect, useState } from 'react'
 
 const storage = window.localStorage
-let lastVersion = storage.getItem('last_version')?.replaceAll('"', '')
-if (lastVersion === undefined) {
-  lastVersion = ''
-}
+const lastVersion = storage.getItem('last_version')?.replaceAll('"', '')
 
 export default function AppVersion() {
   const [appVersion, setAppVersion] = useState(lastVersion)
 
   useEffect(() => {
     window.api.getAppVersion().then((version) => {
+      if (version !== lastVersion) {
+        window.api.logInfo(
+          `Updated to a new version, reloading the MetaMask extension. Old version: ${lastVersion}, new version: ${version}`
+        )
+        window.api.reloadForMM()
+      } else {
+        window.api.logInfo('No new version detected. Wont reload the app')
+      }
```

Comment:
can we do this in the main process instead? these are all backend functions so I think it makes more sense to call them there

Reply:
It is like this because we are using the localStorage, to be only on the main process will need a bit more refactor.
I would need to remove those methods that are not used anymore and also implement the main process one using electron-store instead. 
Doable, but I didn't want to spend too much time on this one tbh. Especially since this already works fine.

Reply:
Pushed a commit to start storing the app version on the config store so in a couple of versions we just need to move the content of the `reloadForMM` to that block in `main.js`  and remove the code on the frontend.

Reply:
Yeah sounds good. I would just hate to have a long lived coupling/dependency of the functioning of mm extension with the presence of this `AppVersion` UI component in the DOM